### PR TITLE
[website] onboarding next iteration

### PR DESCRIPTION
**Pull request by kea14**
PR creation

### CODE REVIEW

File: addons/website/views/website_templates.xml [hunk 1]
```diff
@@ -402,7 +402,7 @@
         </li>
         <li groups="website.group_website_publisher" class="list-inline-item">
             <t t-set="url_return" t-value="url_for('', '[lang]') + '?' + keep_query()"/>
-            <a t-attf-href="/web#action=base.action_view_base_language_install&amp;website_id=#{website.id if website else ''}&amp;url_return=#{url_return}">
```
Comment (qsm-odoo) [inline addons/website/views/website_templates.xml]: Forget about this too: https://drive.google.com/file/d/1BwdRHLCa-e9ouF3LVJ3cAMYH2kVap1JE/view, seems not thought through (and what you did was not what was exactly asked anyway).

File: addons/website/static/src/js/menu/content.js [hunk 2]
```diff
@@ -373,7 +373,18 @@ var MenuEntryDialog = weWidgets.LinkDialog.extend({
...
+            buttons: [
+                {
+                    text: _t("Save"),
+                    classes: 'btn-primary',
+                    close: true,
+                },
+                {
+                    text: _t("Cancel"),
+                    close: true,
+                }
```
[the remark this answers is not in the document]
Comment (kea14) [reply addons/website/static/src/js/menu/content.js]: Indeed this is useless as this is not the prob and I should have removed that, sorry :smile:

File: addons/website/views/website_templates.xml (hunk 1 above)
Comment (kea14) [reply addons/website/views/website_templates.xml]: Fred asked me to do this but anyway I removed it :smile:

File: addons/website/views/snippets.xml [hunk 3]
```diff
@@ -1105,30 +1107,18 @@
                 <a href="#" class="dropdown-item" role="menuitem" data-interval="0">Disable autoplay</a>
             </div>
         </div>
-        <a href="#" class="dropdown-item" data-add-slide="true" data-no-preview="true"><i class="fa fa-plus-circle"/>Add Slide</a>
-        <a href="#" class="dropdown-item" data-remove-slide="true" data-no-preview="true"><i class="fa fa-trash-o"/>Remove Slide</a>
```
[the remark this answers is not in the document]
Comment (kea14) [reply addons/website/views/snippets.xml]: I removed it but https://drive.google.com/file/d/1ZOX25Dx4L78HP8yjTg6pcR5A0dMrB-Nz/view ?

File: addons/website/views/website_templates.xml (hunk 1 above)
Comment (kea14) [reply addons/website/views/website_templates.xml]: I removed `t-if="len(languages) &gt; 1"` but what about this https://drive.google.com/file/d/1KS5r8iAZSnKtk9-7tagw_4hzjPK8awZx/view , should be done or also removed ? ( `d-none d-md-block d-print-block` )

File: addons/website/static/src/js/menu/content.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (kea14) [reply addons/website/static/src/js/menu/content.js]: Concerning buttons translation, I'm not sure on how to fix it, do you have an idea ?

File: addons/website/views/website_templates.xml (hunk 1 above)
Comment (qsm-odoo) [reply addons/website/views/website_templates.xml]: You can handle the mobile stuff. I would not put d-print-block though

File: addons/website/views/snippets.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (qsm-odoo) [reply addons/website/views/snippets.xml]: Ok, I did not see that. You can do it.

**Comment by qsm-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**